php/libraries/Utility.class.inc

```diff
@@ -539,7 +539,7 @@ class Utility
      */
     static function toArray($var)
     {
-        if (!isset($var[0])) {
+        if (!array_key_exists(0, $var)) {
```

Comment:
if thats the reason this function exists then fair, but you still need to check if the `$var` is an array before calling `array_key_exists` as it expects the parameter to be of type `array`